llpc/builder/llpcPipelineState.h

```diff
@@ -181,6 +181,14 @@ class PipelineState final : public Pipeline
     // Get user data nodes
     ArrayRef<ResourceNode> GetUserDataNodes() const { return m_userDataNodes; }
 
+    // Find the resource node for the given set,binding
+    std::pair<const ResourceNode*, const ResourceNode*> FindResourceNode(ResourceMappingNodeType  nodeType,
+                                                                         uint32_t                 descSet,
+                                                                         uint32_t                 binding) const;
+
+    // Get whether we have a converting sampler in the user data nodes.
+    bool GetHaveConvertingSampler() const { return m_haveConvertingSampler; }
```

Comment:
GetHaveConvertingSampler -> HaveConvertingSampler
I think this is more natural.